### PR TITLE
Added new workflow to polish assembly

### PR DESCRIPTION
**Pull request by AnnaSyme**
Added to folder "genome-assembly", in subfolder "polish-with-long-reads"

### CODE REVIEW

**Comment by AnnaSyme** [comment]
ping @mvdbeek  this is ready to go if you wouldn't mind having a look?

**Comment by mvdbeek** [comment]
Looks great, my only suggestion is to the minimap profile parameter to restrict the values users can select (or provide a list of options, if some of them don't make sense)
Screenshot 2023-08-16 at 10 14 37

**Comment by AnnaSyme** [comment]
thanks @mvdbeek this is a good idea. I want to provide a default value but also a list of suggested values. However is this possible? When I do put in the default, but also the suggested values, I can't see them in the drop-down list when I go to run the workflow. If I have no default value, I can see all the suggested values. Is there a way around this?

**Comment by mvdbeek** [comment]
Kind of, the default value only works for optional parameters, so what would work (but is a little clunky) is when you define the possible parameters, you can put the default first.
Screenshot 2023-08-23 at 10 41 21

**Comment by AnnaSyme** [comment]
thanks have done this for these suggested values @mvdbeek

**Comment by mvdbeek** [comment]
Do you need to move Galaxy-Workflow-Assembly_polishing_with_long_reads.ga to Assembly_polishing_with_long_reads.ga ? There are now two workflows in here, and only one has a test.

**Comment by AnnaSyme** [comment]
Thanks @mvdbeek - I've removed the older workflow and re-named the newer one - does that look ok?

File: workflows/genome-assembly/polish-with-long-reads/.dockstore.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: 1.2
+workflows:
+- name: main
+  subclass: Galaxy
+  publish: true
+  primaryDescriptorPath: /Assembly-polishing-with-long-reads.ga
```
Comment (mvdbeek) [inline workflows/genome-assembly/polish-with-long-reads/.dockstore.yml:6]: You'd have to update this as well

Comment (AnnaSyme) [reply workflows/genome-assembly/polish-with-long-reads/.dockstore.yml:6]: Should I update the version?

Comment (mvdbeek) [reply workflows/genome-assembly/polish-with-long-reads/.dockstore.yml:6]: No, it seems your workflow is called `Assembly_polishing_with_long_reads.ga` now, you either need to rename the workflow to `Assembly-polishing-with-long-reads.ga` or change this to `Assembly_polishing_with_long_reads.ga`. I think my preference would be to rename the workflow back to `Assembly-polishing-with-long-reads.ga`

File: workflows/genome-assembly/polish-with-long-reads/Assembly-polishing-with-long-reads.ga [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+    "a_galaxy_workflow": "true",
+    "annotation": "Racon polish with long reads, x4",
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "https://orcid.org/0000-0002-9906-0673",
+            "name": "Anna Syme"
+        }
+    ],
+    "format-version": "0.1",
```
Comment (mvdbeek) [inline workflows/genome-assembly/polish-with-long-reads/Assembly-polishing-with-long-reads.ga:11]:
```suggestion
    "format-version": "0.1",
    "release": "0.1",
```

**Review by mvdbeek — APPROVED** [review]
very cool, I'll merge when the tests are passing!

**Comment by bgruening** [comment]
:tada: :tada: :tada: :tada: :tada: :tada:

**Pull request merged**